### DISCUSSION TITLE
Pixelated slides

### DISCUSSION THREAD

**Discussion by innocenat** [Cimbali/pympress#300] [Q&A] [answered]
Hello!

My situation is that, my Windows laptop is 1080p. I was presenting on a 2160p monitor, and the slides are really, really pixelated. I try looking through pympress.ini to see if there are any render resolution set but can't seem to find it. Am I missing anything?

**Comment by Cimbali** [comment]
You mean the slides on the monitor and not on your laptop, right?

There is no resolution setting in pympress.ini, we take it from the screen resolution. However we’ve had issues before trying to figure out the right size of slides especially with resolution scaling across different OSs.

- Is pympress up to date?
- Can you share the monitor resolution setting for _Scale_ (should be 100% or more)? And the Windows version you’re running?
- And the log file (location in Help>About window) to see if any errors happened?
- You could try to disable page caching by setting `maxpages=0` under `[cache]` in pympress.ini to see if it fixes something − it will make it slow to go through several pages quickly though.

**Comment by innocenat** [comment]
1. I just installed it today from WinGet so I hope so.
2. I didn't check at that time (it was in the meeting room with 80" monitor), but I think it might be at 200% or 250%. Windows 11 22H2.
3. No error in the log file.

I just tried with my external monitor set to 175% and while it's not pixelated it's definitely blurry. So I think it must be something to do with scaling.

Anyway, thank you for the response! Glad I wasn't missing something.

**Reply by Cimbali** [reply]
Ok, I’ll check on a Windows with scaling when I get a chance.

**Reply by innocenat** [reply, marked as answer]
Not sure if I should comment here or on the issues you created, but since I have another access to the monitor in question and found the solution.

Basically it was a meeting room 80" 4K monitor, which default to 300% zoom on Windows. Hence, the blurriness/pixelated. When I changed the scaling to 100% the image is crisp.

While playing with scaling on Windows machine, I think the pympress is NOT considered high-DPI aware on Windows. Hence, Windows report post-scaled resolution to the software, causing the blurriness, as the presenter view also show the blurriness on the UI features when on scaled display.

**Reply by Cimbali** [reply]
Thanks! That’s very useful info.
